src/main/webapp/invalid.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
```

Comment:
I think "already_exists.html" may be the better name because there could be many other reason for it to be invalid. A stretch goal would be adding a parameter for this so that the caller can customize the error message. If you are willing to work on the stretch goal, maybe we should leave it as invalid.html.

Reply:
Understood, I've changed it to "already_exists.html" for now